specification/dimensions/pricingmodel.md

```diff
@@ -0,0 +1,39 @@
+# Pricing Model
+
+Pricing Model indicates the type of unit price applied to a charge at the time it was used or purchased. Pricing Model is commonly used to differentiate between charges at on-demand vs. reduced prices when investigating optimization opportunities, like for commitment-based discount coverage.
+
+The PricingModel column MUST be present. This column is of type String and MUST be one of the allowed values. PricingModel MUST NOT be null or empty if SkuPriceId is not null and MUST be null if SkuPriceId is null. PricingModel MUST be "Commitment-Based Discount" when CommitmentDiscountId is not null. PricingModel MUST be "Dynamic" when pricing is determined by the provider and may change over time, regardless of predetermined agreement pricing. PricingModel MUST be "Other" when a PricingModel is present but none of the current allowed values apply.
+
+## Column ID
+
+PricingModel
+
+## Display name
+
+Pricing Model
+
+## Description
+
+Type of unit price applied to a charge at the time it was used or purchased.
+
+## Content constraints
+
+|    Constraint   |      Value       |
+|:----------------|:-----------------|
+| Column required | True             |
+| Data type       | String           |
+| Allows nulls    | False             |
+| Value format    | list-of-values   |
+
+Allowed values:
+
+| Value                     | Description                                                                                                                                                                                                  |
+| :------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| On-Demand                 | Charges priced at the standard rate for the billing account. This includes any applicable negotiated discounts and volume/tiered pricing, but does not include dynamic or commitment-based discount pricing. |
+| Dynamic                   | Charges priced at a variable rate determined by the provider. This includes any product or service with a unit price the provider can change without notice, like interruptible or low priority resources.   |
```

Comment:
this seems to overlap with on-demand -- you can have dynamic on-demand pricing

Reply:
We discussed that and determined "On-Demand" was the clearest term for the non-committed, non-spot pricing. That said, we also talked about a second level of detail here. We decided to push that to post-1.0 due to time.

What are you defining as "dynamic on-demand pricing"?

Reply:
See Mike's new proposal below and let me know if that meets this requirement @sarahcloudpm ?

Reply:
According to this definition, dynamic is for variable rates that change and on-demand is a standard, predetermined rate, so we should be good. I'll go ahead and resolve this one, but please reopen it if we're missing anything.